internal/runners/activate/activation.go

```diff
@@ -19,6 +20,20 @@ import (
 	"github.com/ActiveState/cli/pkg/projectfile"
 )
 
+type activationError struct {
+	error
+}
+
+func (ae activationError) IsFromActivatedShell() {}
```

Comment:
This feels too specific to our current use-case. Can we name this according to the behaviour it should have, not according to the use-case of the behaviour.